tdrs-backend/tdpservice/parsers/row_schema.py

```diff
@@ -45,6 +48,7 @@ def parse_and_validate(self, line, generate_error):
         if not preparsing_is_valid:
             if self.quiet_preparser_errors:
                 return None, True, []
+            logger.info(f"Preparser error(s) encountered: {[error for error in preparsing_errors]}.")
```

Comment:
Same comment as above, can we downgrade this to debug and/or just get a count of errors instead of listing them all out?